knimin/handlers/ag_search.py

```diff
@@ -49,11 +49,8 @@ def post(self):
                     barcode_info[ag_barcode['barcode']]['ag_info'] = ag_barcode
                     lab_barcode_info = db.get_barcode_details(
                         ag_barcode['barcode'])
-                    plate = db.get_plate_for_barcode(
```

Comment:
This call was accessing to an old plate structure that no longer exists. That plate structure was always empty since the plate mapper was never in place. Apparently, during the first iteration of the plate mapper (quite a while back) some DB changed made it to master but they where never populated.